modules/nickserv.js

```diff
@@ -1,31 +1,48 @@
-var config = null;
-var bot = null;
+let config = null;
+let bot = null;
+let log = null;
 
-module.exports.init = function(b) {
-	bot = b;
+// Only respond to a nickserv notice once.
+let identified = false;
 
-	bot.getConfig('nickserv.json', function(err, conf) {
-		console.log(err);
-		if(err) return;
+module.exports.init = (b) => {
```

Comment:
This still needs to be a function, otherwise `this.log` won't work.

The bot calls init via something quite close to `moduleInit.apply(moduleBot, moduleBot)`

Here's the difference I'm meaning: https://runkit.com/euank/5a10b6c0bb407900127d67bd